16-synthesis.Rmd

```diff
@@ -108,23 +108,23 @@ length(spdeps) # 463
 
 ## Gaps and overlaps {#gaps}
 
-Geocomputation is a big area, so there inevitably gaps in this book.
+Geocomputation is a big area, so there are inevitably gaps in this book.
```

Comment:
Definite typo fix, sorry there are many and many thanks for the fixes to this and more!

Reply:
You are very much welcome and no problem at all.